interpreter/namerd/src/main/scala/io/buoyant/namerd/iface/NamerdInterpreterInitializer.scala

```diff
@@ -48,11 +52,34 @@ case class ClientTlsConfig(commonName: String, caCert: Option[String]) {
   }
 }
 
+case class FailureThresholdConfig(
+  minPeriodMs: Option[Int],
+  threshold: Double,
+  windowSize: Option[Int],
+  closeTimeoutMs: Option[Int]
+) {
+  val defaultConfig = ThresholdConfig()
+  def params: Stack.Params = {
+    val thresholdCfg = for {
+      minPeriod <- minPeriodMs
```

Comment:
This will only build a ThresholdConfig if *all* properties are set.  Instead, I think we want to fall back to defaults for each property.

```
ThresholdConfig(
  minPeriod = minPeriodMs.map(_.millis).getOrElse(DefaultMinPeriod),
  ... etc
)
```